atsign-foundation/at_app_cli
# Overwrite support + README badge fixes

closes #59

**- What I did**

- Rewrote the template_builder.dart copy function to support the `--overwrite` flag.
- Added attribution to the dart sdk, which was used in my rewrite
- Fixed the links on badges (all were pointing to at_app package)

**- How I did it**

Local changes.

**- How to verify it**

Generate any template.
Then change some of the files in lib.
Run again and make sure your changes aren't overwritten by default.
Then run a third time, with `--overwrite` and make sure your changes are overwritten.

**- Description for the changelog**
Overwrite support + README badge fixes
